pr2eus_tutorials/launch/pr2_tabletop.launch

```diff
@@ -2,14 +2,31 @@
   <arg name="run_rviz" default="false" />
   <arg name="publish_objects_tf" default="true" />
   <arg name="launch_tracking" default="false" />
-  <arg name="manager" default="pr2_tabletop_object_detector_nodelet_manager"/>
-  <arg name="machine" default="c2" />
-  <arg name="load_machine" default="false" />
+  <arg name="manager" default="pr2_tabletop_object_detector_nodelet_manager" />
+  <arg name="remote" default="false" />
+  <arg name="load_machine" default="false" unless="$(arg remote)" />
+  <arg name="load_machine" default="true" if="$(arg remote)" />
+  <arg name="machine" default="c2" unless="$(arg remote)" />
+  <arg name="machine" default="localhost" if="$(arg remote)" />
```

Comment:
`default` -> `value`

Reply:
Thank you for your review. 
Fix with https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/459/commits/391e0bd68e2468a2529f47f59d8deeb78c3bbbc3 .

Reply:
`pr2_tabletop.launch` is called by [pr2_tabletop_sim.launch](https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus_tutorials/launch/gazebo/pr2_tabletop_sim.launch) which is gazebo simulator tabletop tutorial. So I've changed the code back from `value` to `default`.

Reply:
this is because you add `remote` arg in this PR.
in that case, you should change `pr2_tabletop_sim.launch` to use `remote` arg.
otherwise, `machine` arg and `remote` arg will conflict, which may be confusing.

Reply:
I suspect not. 
When I execute `pr2_tabletop_sim.launch`, `machine` arg and `remote` arg aren't conflicted. 
This is because `machine` arg in `pr2_tabletop.launch` is always overwritten by that in `pr2_tabletop_sim.launch` no matter which `machine` arg is.
Also, setting`remote` arg true in `pr2_tabletop_sim.launch` might makes users confused since remote access is not related to gazebo simulation.

Reply:
そういう意味ではなく，ややこしいので`remote`を追加するんだったら，そちらに統一，そうじゃないのであれば`machine`を使う，という風に統一してください．
あとの人がこんがらがるので．

Reply:
pr2_tabletop_sim.launch から pr2_tabletop.launch を呼び出すときに，`remote` arg を `true` にして良いということでしょうか．
その場合，procilica, rgb, depth をdecompress してpointcloudを再構成しますでしょうか．
私としては，pr2_tabletop_sim.launch では，`machine` arg を `localhost`に，`load_machine` arg を `true` にするという意味では，[ご指摘なさった部分](https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/459/files/78880de9ec9c6cfab05932ac12e99239e7fb9373#diff-36099b04ef7b0fe95ec8ace80185317d01dfa265f637614eb64819bb0f7baa57R6-R10)は，`remote` arg が true となるのですが，
procilica, rgb, depth をdecompress してpointcloudを再構成する必要性はないと思っています．
その場合，`remote` arg とは別に，`use_sim` arg を新たに作って場合分けするという方法が良いでしょうか．

Reply:
@knorth55 さんに修正して頂き，`use_sim` ではなく`sim` arg を新たに作って場合わけをすることになりました．